scripts/appleseed.package.py

```diff
@@ -190,6 +191,103 @@ def merge_tree(src, dst, symlinks=False, ignore=None):
         raise Error(errors)
 
 
+def create_mac_icon(image, icon_path):
+    icon_set = "appleseed.iconset"
+    safe_make_directory(icon_set)
+
+    for size in (16, 32, 64, 128, 256, 1024, 2048):
+        if size < 1024:
+            name = "icon_{0}x{0}.png".format(size)
+            cmd = ['sips', '-z', str(size), str(size), image, '--out', os.path.join(icon_set, name)]
+            # pipe stderr to stdout to silence verbosity
+            subprocess.check_output(cmd, stderr=subprocess.STDOUT)
+        if size > 16:
+            name = "icon_{0}x{0}@2x.png".format(size/2)
+            cmd = ['sips', '-z', str(size), str(size), image, '--out', os.path.join(icon_set, name)]
+            # pipe stderr to stdout to silence verbosity
+            subprocess.check_output(cmd, stderr=subprocess.STDOUT)
+
+    subprocess.check_call(['iconutil', '--convert', 'icns', icon_set, '-o', icon_path])
```

Comment:
Same question for `iconutil`.

Reply:
I'm not sure the exact version of macOS/Xcode iconutil was introduced,  I think it was Xcode 4.5. which is from 2012.